app/static/js/courseManagement.js

```diff
@@ -1,8 +1,34 @@
-function changeTerm() {
-    $('form').submit();
-};
-
-function formSubmit(el) {
-  $("#termSelector").attr('action', '/manageServiceLearning/' + el);
-  $("#termSelector").submit()
-}
+
+function changeTerm() {
+    $('form').submit();
+};
+
+function formSubmit(el) {
+  $("#termSelector").attr('action', '/manageServiceLearning/' + el);
+  $("#termSelector").submit()
+};
+
+function reviewCourses(el) {
+  var courseID=$(el).attr('data-id');
+  $.ajax({
+    url:"/proposalReview/",
+    type:"POST",
+    data:{"course_id":courseID},
+    success: function(modal_html) {
+      $("#review-modal").html(modal_html)
+      $("#myReview").modal('show')
+    }
+  })
+}
+
+function approve_proposal(el){
+  var courseID=$(el).attr('course-id');

```

Comment:
I would use data attributes here. This works because you can set arbitrary attributes, but those should be in data attributes, really.